stress-ng.c

```diff
@@ -928,6 +929,8 @@ static const stress_help_t help_generic[] = {
 	{ NULL,		"sched-period N",	"set period for SCHED_DEADLINE to N nanosecs (Linux only)" },
 	{ NULL,		"sched-runtime N",	"set runtime for SCHED_DEADLINE to N nanosecs (Linux only)" },
 	{ NULL,		"sched-deadline N",	"set deadline for SCHED_DEADLINE to N nanosecs (Liunx only)" },
+	{ NULL,         "sched-reclaim",        "set reclaim cpu bandwidth for deadline schduler (Liunx only)" },
+
```

Comment:
Can the blank line on line 933 be removed. Thanks!

Reply:
Sure.